plugin/transfer/notifiy.go

```diff
@@ -0,0 +1,73 @@
+package transfer
```

Comment:
also, did you do a git move for this file? or just cut and paste?

Reply:
Also the filename is broken: notifiy -> notify.go

Reply:
> also, did you do a git move for this file? or just cut and paste?

IIRC, parts of it we copied (not cut or moved) from file/notify.   The file plugin will need to continue to implement it's own zone transfer implementation until it's deprecated and removed.